bin/problem.py

```diff
@@ -425,19 +436,17 @@ def statement_samples(p):
 
         testcases.sort()
 
-        if len(testcases) == 0:
```

Comment:
where/why did this go?

Reply:
This moved to `latex.py`: https://github.com/RagnarGrootKoerkamp/BAPCtools/pull/392/files#diff-03fcb90513814b3f133a9ad5bff703ee468f356c0023a6b88380f827529fa54cR44-R47 
I basically merged the two `if`-statements. The function `Problem.statement_samples()` is only used in `latex.create_samples_file()`.